submissions/natashafir/memory-pair-game/index.js

```diff
@@ -0,0 +1,123 @@
+const items = [
+    {
+        id: 0,
+        img_url: './img/colibri.gif'
+    },
+    {
+        id: 1,
+        img_url: './img/hello.gif'
+    },
+    {
+        id: 2,
+        img_url: './img/hi.gif'
+    },
+    {
+        id: 3,
+        img_url: './img/inlove.gif'
+    },
+    {
+        id: 4,
+        img_url: './img/love.gif'
+    },
+    {
+        id: 5,
+        img_url: './img/play.gif'
+    },
+    {
+        id: 6,
+        img_url: './img/sleep.gif'
+    },
+    {
+        id: 7,
+        img_url: './img/space.gif'
+    }
+];
+
+let choosedCards = [];
+const sectionMemory = document.getElementById('section__memory');
+let hasFlippedCard = false;
+let boardLocked = false;
+let firstCard, secondCard;
+
+function shuffle(array) {
+    for (let i = array.length - 1; i > 0; i--) {
+        let j = Math.floor(Math.random() * (i + 1));
+        [array[i], array[j]] = [array[j], array[i]];
+    }
+    return array;
+}
+
+function createCardItem(item) {
+    const cardWrapper = document.createElement('div');
+    const imgBack = document.createElement('img');
+    const imgFront = document.createElement('img');
+    cardWrapper.classList.add("memory-card"); //class for CSS
```

Comment:
1. redundant comment
1. if you use single quotes in all your js file, why double used here ? 
```suggestion
    cardWrapper.classList.add('memory-card'); 
```